common/src/main/java/com/wynntils/commands/WynntilsCommand.java

```diff
@@ -214,9 +214,12 @@ private int reloadCaches(CommandContext<CommandSourceStack> context) {
                                 .withStyle(ChatFormatting.YELLOW),
                         false);
 
+        Managers.Url.loadUrls();
```

Comment:
I think it is better to say something "positive" about this instead of something "negative" about the removed code, e.g.
`// This reloads all URLs, and will then trigger a re-download by the DownloadManager`